crates/amaru/src/bin/amaru/cmd/import_ledger_state.rs

```diff
@@ -82,6 +88,14 @@ pub struct Args {
         default_value_t = NetworkName::Preprod,
     )]
     network: NetworkName,
+
+    #[arg(
+        long,
+    )]
+    import_vector: bool,
+
+    #[arg(long, value_name = "DIR")]
+    pparams_dir: Option<PathBuf>,
```

Comment:
These feel shoehorned as they trigger an action completely separate from the import-ledger-state command. It seems to me that this would be better done in a test context? A bit similar to how we do the snapshots tests: https://github.com/pragma-org/amaru/blob/main/crates/amaru/tests/summary.rs